spec/factories/users.rb

```diff
@@ -21,6 +21,24 @@
     linkedin_url { Faker::Internet.url }
   end
 
+  factory :active_user, class: User do
```

Comment:
Instead of creating a whole new factory here, please add a trait to `skinny_user`. The plan is to eventually replace all user factories with skinny_user, and then rename skinny_user to just `user`